### ISSUE TITLE
[Terminal> Command palette]: Screen Reader is not announcing Suggestion name when user navigate to Command Palette suggestions using up/down arrow keys.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12065]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+ Shift + p'.
3. Open Narrator using 'Win + Ctrl + Enter' keys.
4. Now press up/down arrow keys and observe the issues.

**User Experience:**
Screen Reader users UX will not be good as screen reader is not announcing suggestion while navigate to them.

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Screen Reader is not announcing Suggestion name when user navigate to Command Palette suggestions using updown arrow keys..zip](https://github.com/microsoft/terminal/files/7787291/Screen.Reader.is.not.announcing.Suggestion.name.when.user.navigate.to.Command.Palette.suggestions.using.updown.arrow.keys.zip)

### Expected Behavior

Screen Reader should announce Suggestion name when user navigate to Command Palette suggestions using up/down arrow keys.

### Actual Behavior

Screen Reader is not announcing Suggestion name when user navigate to Command Palette suggestions using up/down arrow keys.

**Observation:** NVDA announce 'blank' and JAWS remain silent.
